client/my-sites/stats/activity-log/index.jsx

```diff
@@ -613,10 +615,14 @@ export default connect(
 		const requestedRestoreId = getRequestedRewind( state, siteId );
 		const requestedBackupId = getRequestedBackup( state, siteId );
 		const logRequestQuery = getActivityLogQuery( { gmtOffset, startDate, timezone } );
+		const restoreStatus = get( state, [ 'rewind', siteId, 'rewind' ], false );
```

Comment:
note that we already have `getRewindState( siteId )` and it always returns a valid object.

we could do something like this…

```js
const rewindStatus = getRewindState( state, siteId );
const restoreStatus = rewindStatus.rewind && rewindStatus.rewind.status;
const enableRewind = (
	'uninitialized' !== rewindStatus.state &&
	! ( restoreStatus === 'queued' || restoreStatus === 'running' )
);
```

That is, make sure we hav an initialized `rewindState` and then anything other than a running rewind means we won't block. I'm not sure the `include()` and `get()`s add much for clarity here.